src/tests/execute/approve_ask_tests.rs

```diff
@@ -280,6 +280,7 @@ mod approve_ask_tests {
               \"denom\": \"base_1\",
               \"total_supply\": \"1000\",
               \"marker_type\": \"restricted\",
+              \"allow_forced_transfer\": false,
```

Comment:
I'm not exactly sure why this started causing issues since the only dependency that changed was the `cosmwasm-storage`  removal